internal/host/logscmd.go

```diff
@@ -44,3 +56,29 @@ func (i *logsCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models.St
 
 	return []*models.Step{step}, nil
 }
+
+func (i *logsCmd) getMastersIPsInHostCluster(ctx context.Context, host *models.Host) ([]string, error) {
+
+	var cluster models.Cluster
+	if err := i.db.First(&cluster, "cluster_id = ?", host.ClusterID).Error; err != nil {
```

Comment:
Seems this is wrong.  The ID for cluster is "id" - not "cluster_id"

Reply:
The object is `host`, `host.ID` will get the id of the host, I want the id of the cluster which the host is part of it.

Reply:
No.  You are getting cluster here

Reply:
fixed.